remotespark/livyclientlib/livysession.py

```diff
@@ -75,7 +75,10 @@ def create_sql_context(self):
 
```

Comment:
In line 68 of this file, you should also add the SparkContext as:
`self.ipython_display.writeln("Creating SparkContext as 'sc'")`